script.check.theme/addon.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.check.theme" name="Check MP3 Theme" version="0.0.2" provider-name="manfeed">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.python.library" library="default.py"/>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">MP3 theme check addon</summary>
+		<description lang="en_GB">Script for skinners to check if a mp3 theme exists.</description>
+		<language></language>
```

Comment:
please remove the language tag as this script doesn't exactly provide any content